scriptmodules/supplementary/skyscraper.sh

```diff
@@ -0,0 +1,680 @@
+#!/usr/bin/env bash -x
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="skyscraper"
+rp_module_desc="Scraper for EmulationStation by Lars Muldjord"
+rp_module_licence="GPLv3.0 https://raw.githubusercontent.com/muldjord/skyscraper/master/LICENSE"
+rp_module_section="opt"
+
+function depends_skyscraper() {
+    getDepends qt5-default p7zip-full
+}
+
+function sources_skyscraper() {
+    local latest_ver
+    latest_ver=$(latest_ver_skyscraper)
+
+    gitPullOrClone "$md_build" "https://github.com/muldjord/skyscraper" "$latest_ver"
+}
+
+function build_skyscraper() {
+    qmake
+    make
+    md_ret_require="$md_build/Skyscraper"
+}
+
+function install_skyscraper() {
+    md_ret_files=(
+    'Skyscraper'
+    'LICENSE'
+    'README.md'
+    )
+}
+
+# Purge all Skyscraper caches
+function _purge_skyscraper() {
+    find "$configdir/all/skyscraper/dbs" -maxdepth 1 -mindepth 1 -type d -print0 -exec rm -fr {} \;
```

Comment:
could just remove the dbs folder ? it's recreated I assume ?

Reply:
I think so. I'll test and see if there's any error reported.

Reply:
The folder is recreated, but there are example files in the folder that should not be deleted. It might even be worth considering not deleting the platform subfolders themselves, since the user might have created custom dbs/[platform]/priorities.xml files for each platform. So basically a purge for all systems should remove only dbs/[platform]/db.xml files and all folders in dbs/[platform]/[folders]. Same goes for purging a single platform.

Reply:
OK, I'll modify the `purge_..` functions to take this into account. I knew something about the `priorities.xml`, but though it's a global file.